templates/tom_targets/partials/target_data.html

```diff
@@ -0,0 +1,34 @@
+{% load tom_common_extras targets_extras %}
+<dl class="row">
+  {% for target_name in target.names %}
+    {% if forloop.first %}
+      <dt class="col-sm-6">Names</dt>
+    {% else %}
+      <dt class="col-sm-6">&nbsp;</dt>
+    {% endif %}
+    <dd class="col-sm-6">{{ target_name }}</dd>
+  {% endfor %}
+  {% for key, value in target.as_dict.items %}
+  {% if value and key != 'name' and key != 'epoch' and value != 'SIDEREAL' %}
+  <dt class="col-sm-6">{% verbose_name target key %}</dt>
+  <dd class="col-sm-6">{{ value|truncate_number }}</dd>
+  {% endif %}
+  {% endfor %}
```

Comment:
Maybe it's better to just list RA and dec. explicitly here, instead of looping through everything and then excluding most of them. Also for the coordinates we should use more than the default of 4 decimal places.